spec/lib/appsignal/check_in_spec.rb

```diff
@@ -0,0 +1,303 @@
+describe Appsignal::Heartbeat do
+  let(:err_stream) { std_stream }
+
+  after do
+    Appsignal.instance_variable_set(:@heartbeat_constant_deprecation_warning_emitted, false)
+  end
+
+  it "returns the Cron constant calling the Heartbeat constant" do
+    silence { expect(Appsignal::Heartbeat).to be(Appsignal::CheckIn::Cron) }
+  end
```

Comment:
We can remove this kind of  #aliasing once merged into the develop branch. I've linked it on the tracking issue #667 .